db/migrate/20180920200522_create_join_table_portfolio_portfolio_item.rb

```diff
@@ -0,0 +1,12 @@
+class CreateJoinTablePortfolioPortfolioItem < ActiveRecord::Migration[5.1]
+  def change
+    create_join_table :portfolios, :portfolio_items do |t|
+      t.index [:portfolio_id, :portfolio_item_id], name: 'index_items_on_portfolio_id_and_portfolio_item_id'
+      t.index [:portfolio_item_id, :portfolio_id], name: 'index_items_on_portfolio_item_id_and_portfolio_id'
```

Comment:
Out of curiosity what was the original names that didn't work.  The ones you created look exactly like the defaults, unless I'm missing something.

Reply:
Also do you need both indexes?  That is you should only need one unless there are cases where either of the two ids will be specified.

Reply:
Here is one of the originals: `index_portfolio_items_portfolios_on_portfolio_id_and_portfolio_item_id` - I just pulled out the redundant 'portfolio' text